tools/node-lint-md-cli-rollup/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "node-lint-md-cli-rollup",
+  "description": "remark packaged for node markdown linting",
+  "version": "1.0.0",
+  "devDependencies": {
+    "rollup": "^0.55.5",
+    "rollup-plugin-commonjs": "^8.0.2",
+    "rollup-plugin-json": "^2.3.0",
+    "rollup-plugin-node-resolve": "^3.0.0"
+  },
+  "dependencies": {
+    "markdown-extensions": "^1.1.0",
+    "remark": "^8.0.0",
+    "remark-lint": "^6.0.0",
+    "remark-preset-lint-node": "^1.0.3",
+    "unified-args": "^4.0.0",
+    "unified-engine": "^5.1.0"
+  },
+  "scripts": {
+    "build": "rollup -c",
+    "build-node": "npm run build && cp dist/* .."
```

Comment:
The `lint-md-build` is removed, can you add it back with a command that runs this?